package.json

```diff
@@ -313,7 +313,7 @@
     "grunt-npm": "0.0.2",
     "jasmine-core": "^2.3.4",
     "json3": "^3.3.2",
-    "karma-browserify": "^5.0.1",
+    "karma-browserify": "^5.0.2",
```

Comment:
The change @ karma-browserify won't be a patch version, too because it will break with all karma versions < than the new one.